### DISCUSSION TITLE
Set randomSeed to Webots World for correct reproducibility

### DISCUSSION THREAD

**Discussion by angel-ayala** [cyberbotics/webots#6085] [Q&A]
Hi, I'm trying to set the seed value for random number generator and as mentioned in the documentation that value must be set in the `randomSeed` field from the `WorldInfo` Node.

I know that using the `Supevisor` I can change the field values in any Node during runtime. Can this be done?
Or which is the correct way to set that value?

In Python I'm currently using this line of code to set a `seed` value.
`self.__robot.getFromDef('World').getField('randomSeed').setSFInt32(0 if seed is None else seed)`

**Comment by ygoumaz** [comment]
Could you please post this question to https://robotics.stackexchange.com/ with the `webots` tag, as it may get more visibility and help more users in the future?
